src/bin/hexyl.rs

```diff
@@ -45,6 +45,13 @@ fn run() -> Result<(), Box<dyn std::error::Error>> {
                 .value_name("N")
                 .help("Skip first N bytes"),
         )
+        .arg(
+            Arg::with_name("block_size")
+                .long("block-size")
+                .takes_value(true)
+                .value_name("N")
+                .help("Sets `block` unit to N"),
```

Comment:
minor: *"… unit to N bytes"*?.

Reply:
Good catch. I'll fix this.

Actually, this raises an interesting question to me.  It could be useful to interpret a byte offset here instead of just straight bytes, since often block sizes are multiples of `kb`. That can come later, if necessary, but it seems just as easy to do now. What do you think of applying `parse_byte_offset` here?

Reply:
That makes total sense, yes :+1: In that case, the help text can either stay this way or maybe rename `N` to `SIZE` and then:

*"Set the size of the \`block\` unit to SIZE. Examples: --block-size=1024, --block-size=4kB"*

Reply:
Done. The only modification I made to your suggestion is to use indicative tense ("Sets") instead of imperative ("Set"), since that's the tense most of the options use.